apps/api/src/server/applications/zoom-level/__tests__/get-zoom-map.test.js

```diff
@@ -33,3 +44,29 @@ describe('Get zoom map', () => {
 		]);
 	});
 });
+
+test('tests if cache is working', async () => {
```

Comment:
I think these can be a little more descriptive. "Tests if cache is working" does not say exactly what the cache need to be doing